docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md

```diff
@@ -1,8 +1,8 @@
-# Transfer your app from `appc` CLI to `ti` CLI
+# Transfer your app from Appcelerator CLI to Titanium CLI
 
 ## Introduction
 
-Axway announced that they will hand over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
+Axway handed over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
```

Comment:
Axway transferred Titanium to the community in ...

Get rid of the `<b>` and replace the `!</b>.` with `,`. You don't want a period to follow a exclamation point.

Reply:
typos are fixed. thank yu